src/dlgprefvinyldlg.ui

```diff
@@ -593,65 +593,65 @@
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
-       <layout class="QHBoxLayout" name="horizontalLayout_2">
+       <widget class="QLabel" name="textLabel1_3">
+        <property name="text">
+         <string>Turntable Preamp</string>
```

Comment:
Use whole words, and in other parts of the code we say "boost", so maybe "Turntable input signal boost"